packs/data/kingmaker-bestiary.db/svetlana.json

```diff
@@ -137,38 +151,70 @@
                 "negateBulk": {
                     "value": "0"
                 },
+                "potency": {},
+                "potencyRune": {
+                    "value": 0
+                },
                 "preciousMaterial": {
-                    "value": null
+                    "value": ""
                 },
                 "preciousMaterialGrade": {
-                    "value": null
+                    "value": ""
                 },
                 "price": {
-                    "value": {}
+                    "value": {
+                        "gp": 2
+                    }
+                },
+                "propertyRune1": {
+                    "value": ""
+                },
+                "propertyRune2": {
+                    "value": ""
+                },
+                "propertyRune3": {
+                    "value": ""
+                },
+                "propertyRune4": {
+                    "value": ""
                 },
                 "quantity": 1,
+                "resiliencyRune": {
+                    "value": ""
+                },
                 "rules": [],
                 "size": "med",
-                "slug": null,
+                "slug": "leather-armor",
                 "source": {
-                    "value": ""
+                    "value": "Pathfinder Core Rulebook"
+                },
+                "speed": {
+                    "value": 0
                 },
                 "stackGroup": null,
+                "strength": {
+                    "value": 10
+                },
                 "traits": {
                     "rarity": "common",
                     "value": []
                 },
                 "usage": {
-                    "value": "held-in-one-hand"
+                    "value": "wornarmor"
                 },
                 "weight": {
-                    "value": "-"
+                    "value": "2"
                 }
             },
-            "type": "equipment"
+            "type": "armor"
         },
         {
-            "_id": "X6qxLgkALLxdJTTq",
+            "_id": "IP2DHIvoqbsWaRVU",
+            "flags": {
+                "pf2e": {
+                    "linkedWeapon": "KV65DhNy4Axid8kc"
+                }
+            },
```

Comment:
```suggestion
```
This gives me pause because the Cleaver isn't a properly defined base weapon anywhere else.